templates/app.json.erb

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "<%=app_name%>",
+  "scripts": {
+  },
+  "env": {
+    "AWS_ACCESS_KEY_ID":        { "required": false },
+    "AWS_REGION":               { "required": false },
+    "AWS_SECRET_ACCESS_KEY":    { "required": false },
+    "S3_BUCKET_NAME":           { "required": false },
+    "DEPLOY_TASKS":             { "required": false },
+    "TZ":                       { "required": false },
+    "HEROKU_APP_NAME":          { "required": true },
+    "LANG":                     { "required": true },
+    "RACK_ENV":                 { "required": true },
+    "RAILS_ENV":                { "required": true },
+    "RAILS_SERVE_STATIC_FILES": { "required": true },
+    "SECRET_KEY_BASE":          { "required": true }
+  },
+  "formation": {
+    "worker": { "quantity": 1 },
+    "web":    { "quantity": 1 }
+  },
+  "addons": [
+    "heroku-postgresql",
+    "scheduler"
```

Comment:
No, tady ten scheduler bych nahradil heroku-redis, pro sidekiq-cron, ale otázka je, jestli to dávat jako default. Všechny naše produkty budou mít nějaký cron?